jubilant/_juju.py

```diff
@@ -654,7 +654,7 @@ def wait(
             prev_status = status
             status = self.status()
             if status != prev_status:
-                logger.info('wait: status changed:\n%s', status)
+                logger.info('wait: status changed:\n%s', _status_diff(prev_status, status))
```

Comment:
Do we always exit hard if `juju status` fails?

I've only seen that when a model is removed, but I wonder about the potential of other corner cases.

Reply:
Well, if `juju status` fails we'll raise a `CLIError`. I think that's okay, because like you say, it should only happen if the model's torn down (which will normally only happen after the test finishes). If this is a problem we can always revisit.